README.md

```diff
@@ -1,68 +1,120 @@
-
-## Main features
-- upload video
+## Features
+- create a project<sup>[1](#project)</sup>
+- list all projects
+- retrieve project details
+- delete project
+- duplicate project
 - edit video:
     * trim
     * rotate
     * scale
     * crop
-- manage video projects:
-    * retrieve
-    * list
-    * create
-    * duplicate
-    * delete
-- capture and save thumbnails for preview and timeline
-- upload custom thumbnail for preview
-- get thumbnails
+- capture a thumbnails for timeline<sup>[2](#timeline)</sup>
+- capture a thumbnail for a preview at a certain position of the video, with optional crop and rotate params
+- upload a custom image file for a preview thumbnail
+- get thumbnails files
+- get video file
 - stream video
 
-### Installing
+<a name="project">1</a>: `project` it's a record in db with metadata about video, thumbnails, version, processing statuses, links to files and etc.   
+<a name="timeline">2</a>: `timeline` is a display of a list of pictures in chronological order. Useful if you build a UI.
+
 
+## Installation & Run
 These services must be installed, configured and running:
 
  * Python (>= 3.6)
  * FFmpeg
  * MongoDB 
  * RabbitMQ (celery backend)
 
-After required services were installed and running, 
-you will need to clone the repo and install python dependencies:
+After required services were installed and started, 
+you can proceed with a video server installation.
 
-NOTE: use [virtualenv](https://docs.python.org/3/library/venv.html) and [pip](https://pypi.org/project/pip/) to install python modules.
+
+### Installation for development
+NOTE: Use [virtualenv](https://docs.python.org/3/library/venv.html) and [pip](https://pypi.org/project/pip/) to install python modules.
 
 ```
 # clone project
-git clone git@github.com:superdesk/video-server.git
+git clone https://github.com/superdesk/video-server.git
+
+# install video server for development
+# NOTE: your virtualenv must be activated
+pip install -e video-server/[dev]
+```
+
 
-# install python dependencies
-pip3 install -r requirement.txt
+### Run video server for development
+Video server consists from two main parts: http api and celery workers.  
 
-# run gunicorn process for http api and celery for delayed jobs
-honcho start
+For starting an http api dev server:
+1. Set `FLASK_ENV` env variable to `development`:
 ```
+export FLASK_ENV=development
+```
+2. Create `app.py` file:
+```python
+import os
+
+from videoserver.app import get_app
 
+if __name__ == '__main__':
+    host = '0.0.0.0'
+    port = int(os.environ.get('PORT', '5050'))
+    app = get_app()
+    app.run(host=host, port=port)
+``` 
+3. Run `app.py`  
+`python app.py`
 
-## Running the tests
+For starting a celery workers:
+1. <a name="worker">Create</a> `worker.py` file:
+```python
+import logging
+from videoserver.app import get_app
 
-Just run from the project's root:
+logger = logging.getLogger(__name__)
+celery = get_app().celery
+``` 
+2. Run workers  
+`celery -A worker worker`
```

Comment:
would be good to put such `worker.py` file in videoserver package and then you can start celery like:
```
celery -A videoserver.worker worker
```
so we don't have to copy those files around